### DISCUSSION TITLE
Integration with n8n and Workflow Automation

### DISCUSSION THREAD

**Discussion by leminhhai** [makeplane/plane#2648] [Ideas]
Firstly, I would like to express my appreciation for the proactive and seamless integration capabilities offered by n8n. With n8n, we have found it incredibly easy to integrate plane.so with various other software tools that we currently utilize, such as Freshdesk, Freshsales, FeatureOS, AnnounceKit, and Chameleon. This integration has allowed us to link issues and customer feedback, as well as update the status of tasks.

One of the key advantages of integrating plane.so with n8n is the ability to update task priorities based on calculated formulas, similar to the approach taken by GitLab Product. By leveraging this functionality, we can prioritize features based on metrics such as retention revenue and growth revenue. This data-driven approach ensures that our product team focuses on features that provide maximum value to our customers and generate revenue for our company.

Furthermore, the integration between plane.so and n8n enables us to have a more comprehensive and data-driven view of our product. By linking customer feedback and issues from various software tools, we can gather valuable insights and make informed decisions. This integration not only streamlines our workflow but also empowers our product team to prioritize tasks effectively and align them with the overarching business objectives.

In conclusion, I strongly believe that integrating plane.so with n8n and leveraging workflow automation capabilities will significantly enhance our product development process. The ability to link issues and customer feedback, update task statuses, and calculate task priorities based on predefined formulas will enable us to create a more data-driven and customer-centric product.

Thank you for considering this feedback. We look forward to seeing plane.so further evolve with enhanced integration capabilities and workflow automation.

**Comment by bitsByB** [comment]
I second this comment, it would be amazing to have an n8n integration, both for the users and the growth of Plane (as n8n is exploding in popularity - 65k github stars right now)
To make it happen, Plane's dev could start building a community node (https://docs.n8n.io/integrations/community-nodes/build-community-nodes/ ) that would be then moved into the core n8n nodes library.
That's the fastest way to have those integrations going for apps.
Clickup and Taiga are already fully integrated in n8n.
Looking forward Plane team, thanks for the hard work and great app !

**Comment by AncientNimbus** [comment]
To those who are looking for a temporary solution, I have managed to find a work around to those who self-hosted both Plane and n8n.
- Added both containers to the same Docker network for direct communication
- Used direct IP address and port (http://your-n8n-instance-ip:PORT/webhook-test/xxx) in webhook URL instead of domain name
- Set authentication to "None" 

Do be mindful that this setup is recommended for local network only as my setup is intended for closed networking.

**Comment by hamzatmirza** [comment]
I'm willing to create a N8N node by integrating the Public APIs and Webhook. Can the dev team confirm if anyone else is already working on it ?

**Comment by Prashant-Surya** [comment]
@hamzatmirza we have it on our roadmap but it's not an immediate priority for us right now.

**Reply by zippypaul** [reply]
hi @Prashant-Surya 
Great to hear that it's on there at least.  We use both Plane.SO and n8n and yeah we're also hanging out for this integration.
any updates now in Feb 2026 on this?

**Reply by Prashant-Surya** [reply]
@zippypaul we started building a PoC for n8n integration and it's looking good. 

This will soon be available by Q1

**Reply by zippypaul** [reply]
oh great, if you want someone to be your guinee pig, hit us up, we'd be keen.

**Reply by Clauber** [reply]
Is it available now? 👀
